commands/activity.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'node:fs/promises'
+import { Tail } from 'tail'
+import { paths } from '../lib/paths.js'
+import { parseLog, formatLog } from '../lib/log.js'
+
+const formatLogLine = line => {
+  const { date, text } = parseLog(line)
+  const { message } = JSON.parse(text)
+  return formatLog(message, date)
+}
+
+const maybeCreateActivityFile = async () => {
```

Comment:
Is it time to extract a shared helper for creating a new state file if it does not exist yet?

Reply:
I don't think so, to me it's not complex enough to benefit from DRY